aten/src/ATen/cuda/cub-RadixSortPairs-f16s.cu

```diff
@@ -0,0 +1,11 @@
+#include <ATen/cuda/cub-RadixSortPairs.cuh>
+
+namespace at::cuda::cub::detail {
+
+AT_INSTANTIATE_SORT_PAIRS(c10::BFloat16, 8)
```

Comment:
So these are we all separated out into one instantiation per CUDA file in a previous PR to improve compile time @voidbag . Re-aggregating them like this might be partially responsible for the compile time regression since it makes the instatiation less parallel.

Since we have a lot of inationation of similar bit width now, I am not sure if will increase compile size if we do so btw.